tests/sample-computed-fields/computed-field-types/goes-well-with-color.js

```diff
@@ -2,6 +2,7 @@ exports.type = '@cardstack/core-types::boolean';
 
 exports.compute = async function(model, { color }) {
   let otherFoods = await model.getRelated('goes-well-with');
+  if (!otherFoods) { return false; }
```

Comment:
note: the seed model for these tests did not necessarily have relationships, needed to add a guard here.